### PR TITLE
Add badges in README and update links

### PR DESCRIPTION
**Pull request by justeen35**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add badges in README and update links
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Look at the README :)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -28,6 +32,6 @@ Just make sure to follow our [contribution guidelines][contribution-guidelines].
 This module is released under the [Academic Free License 3.0][AFL-3.0] 
 
 [report-issue]: https://github.com/PrestaShop/PrestaShop/issues/new/choose
-[prestashop]: https://www.prestashop.com/
-[contribution-guidelines]: https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/project-modules/
+[prestashop]: [https://www.prestashop.com/](https://www.prestashop-project.org/)
+[contribution-guidelines]: https://devdocs.prestashop.com/8/contribute/contribution-guidelines/project-modules/
```
Comment (mflasquin) [inline Readme.md]:
```suggestion
[contribution-guidelines]: https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/project-modules/
```

**Review by mflasquin — DISMISSED** [review]

File: Readme.md (hunk 1 above)
Comment (justeen35) [reply Readme.md]: Fixed, thank you

**Comment by justeen35** [comment]
@mflasquin All good

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Looks like PHPStan is finding a problem however unrelated to this PR 🤔

**Comment by M0rgan01** [comment]
@matks Since the switch to Prestashop v8, the method that phpstan provides no longer exists.

> 80     Call to an undefined static method Configuration::getInt().  

https://github.com/PrestaShop/PrestaShop/blob/e57ae6124e0ad897ff74078b74d7a72bbf835b54/classes/Configuration.php#L257-L263 in 1.7.8.x branch

Need another PR to fix the problem.

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @justeen35 :)

**Pull request merged**